--- samsung-backlight-kmod-svn20101109/Makefile.orig	2010-11-09 16:35:21.000000000 +0300
+++ samsung-backlight-kmod-svn20101109/Makefile	2010-11-09 20:25:34.984655055 +0300
@@ -1,5 +1,6 @@
 obj-m	:= samsung-backlight.o
 
+INST_DIR  ?= /lib/modules/$(shell uname -r)/extra/samsung-backlight/
 KERNELDIR ?= /lib/modules/$(shell uname -r)/build
 PWD       := $(shell pwd)
 
@@ -10,3 +11,6 @@
 	rm -f *.o *~ core .depend .*.cmd *.ko *.mod.c
 	rm -f Module.markers Module.symvers modules.order
 	rm -rf .tmp_versions Modules.symvers
+
+install:
+	install -D -m 644 -p samsung-backlight.ko $(INST_DIR)/samsung-backlight.ko
